server/startup/initialData.js

```diff
@@ -106,6 +106,22 @@ Meteor.startup(function() {
 			}
 		}
 
+		if (typeof process.env.INITIAL_USER === 'string' && process.env.INITIAL_USER.length > 0) {
+			try {
+				const initialUser = JSON.parse(process.env.INITIAL_USER);
+
+				if (!initialUser._id) {
+					console.log('No _id provided; Ignoring environment variable INITIAL_USER'.red);
+				} else if (!RocketChat.models.Users.findOneById(initialUser._id)) {
+					console.log('Inserting initial user:'.green);
+					console.log(JSON.stringify(initialUser, null, 2).green);
```

Comment:
I'm not sure it's a good idea to print the environment variable value. it can have some sensitive information that could be send to external services that manage the rocket.chat instance